mautic/mautic
# Fix date time field display issue

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.3.0
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:
Some datetime are not displayed according to user local timezone : 
- in contact details
- in contact audit logs details



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. create a new customfield of type datetime for a contact
![Screenshot 2021-03-02 at 15 24 14](https://user-images.githubusercontent.com/79137416/109662537-759ef180-7b6b-11eb-8eea-32df50483c8a.png)

3. edit a contact and set the field
![Screenshot 2021-03-02 at 15 28 41](https://user-images.githubusercontent.com/79137416/109663098-04137300-7b6c-11eb-8d33-254d060bfbfd.png)

4. see that your customfield is displayed in accordance with your local timezone (mine is Europe/Paris) in contact details and in auditlog 
![Screenshot 2021-03-02 at 15 37 10](https://user-images.githubusercontent.com/79137416/109664442-6b7df280-7b6d-11eb-98c8-c2728249d8f7.png)

![Screenshot 2021-03-02 at 15 38 47](https://user-images.githubusercontent.com/79137416/109664452-6e78e300-7b6d-11eb-82d5-a378f8ab0426.png)
